ibc/handle_message_test.go

```diff
@@ -0,0 +1,210 @@
+package ibc
```

Comment:
This is the testing of the message handling and would like some feedback on any other tests to go in here/expand on any of the current ones & any tests to go in persistence to test the Postgres side of things. Just high level what would be nice to see

@Olshansk @dylanlott

Reply:
@h5law From a unit testing perspective, this is pretty extensive and you did a good job. 

That said, I understand where your concern / lack of confidence comes from and unfortunately we'll have to live with it until we've made some progress. I have two suggestions:

1. Pipe this whole `git diff` through ChatGPT and say 

> You are a senior engineer with experience in protocol development, and expertise in the IBC protocol. Review the tests I have implemented in `handle_message_test` and provide some additional test cases that I have not accounted for yet.

2. Though we don't have the infra / CLI to test this E2E, do the following:
2.1 Create a placeholder `ibc.feature` and write out (in plain English) what kind of tests you want to have in the future
2.2 Create a new ticket to implement them later and leave a TODO